openrndr-math/src/main/kotlin/org/openrndr/math/CatmullRom.kt

```diff
@@ -96,11 +130,22 @@ class CatmullRom2(val p0: Vector2, val p1: Vector2, val p2: Vector2, val p3: Vec
     }
 }
 
+/**
+ * Calculates the 2D Catmull–Rom spline for a chain of points and returns the combined curve.
+ *
+ * For more details, see [CatmullRom2].
+ *
+ * @param points The [List] of 2D points where [CatmullRom2] is applied in groups of 4.
+ * @param alpha The so-called "tension" of the curve.
+ *      Use `0.0` for the uniform spline, `0.5` for the centripetal spline, `1.0` for the chordal spline.
+ * @param loop Whether or not to connect the first and last point so it forms a closed shape.
+ */
```

Comment:
Maybe mention that alpha has no effect for points evenly distributed on a circle. I'm not even sure what the real condition is, but when I used the vertices of a pentagon or hexagon, alpha did nothing and I thought it was broken :-)

Reply:
Is there any particular phrasing you'd prefer?

Reply:
```alpha has no observable effect when the points are positioned equally spaced on an imaginary circle```

Reply:
Should this be added to other dimensions of the CatmullRom functions as well?

Reply:
as in, I'm unsure of how well it applies to 1D splines but a similar effect can probably be encountered with a sphere in 3D?

Reply:
The only reason I know about this ones specific case is because in my first test of CatmullRom I placed my points in a pentagon and alpha was doing nothing, so I thought it was broken but it was only due to my specific layout. I'm not sure under what conditions alpha has no effect, sorry. 

Here an improved version (because it mentions the shape must be closed)

    alpha has no observable effect for closed shapes formed by the vertices of a regular polygon

Reply:
I would say that it also happens in 3D, as you could place the regular polygon floating in 3D.

Reply:
In 3D, I do not know if a condition is that all points are in the same plane, or is it possible under other circumstances too.

Reply:
We could be vague and say that 

    under some circumstances alpha has no observable effect, for example when forming closed shapes formed by the vertices of a regular 2D polygon

This would apply to 2D and 3D